doc/source/user_guide/io.rst

```diff
@@ -275,6 +275,14 @@ parse_dates : boolean or list of ints or names or list of lists or dict, default
 infer_datetime_format : boolean, default ``False``
   If ``True`` and parse_dates is enabled for a column, attempt to infer the
   datetime format to speed up the processing.
+
+  .. deprecated:: 1.5.2
+   ``infer_datetime_format`` be deprecated (as a strict version of it will become the default);
+    an easy workaround for non-strict parsing be clearly documented.
+
+   Currently, the only way to ensure consistent parsing is by explicitly passing
```

Comment:
could you copy the deprecation message from https://pandas.pydata.org/docs/dev/reference/api/pandas.to_datetime.html please

Reply:
yup

Reply:
should I be directly using it as it is?

Reply:
yes